src/btf.h

```diff
@@ -24,6 +26,10 @@ class BTF
 
 public:
   BTF();
```

Comment:
Any other callers of this? I'd rather BTF() only have one constructor so nobody forgets to pass `BPFtrace*`

Reply:
Unfortunately yes, it's called from `BPFfeature` which doesn't have access to `BPFtrace` ([bpffeature.cpp:199](https://github.com/iovisor/bpftrace/blob/master/src/bpffeature.cpp#L199))